src/selectors/layers.test.js

```diff
@@ -35,7 +35,7 @@ describe('Selectors', () => {
 
       nodes.forEach(node => {
         // we don't need to check y/height positions if the layer isn't there.
-        if (node.layer === null) {
+        if (!node.layer || node.layer === null) {
```

Comment:
If this is due to `undefined` then check for that instead, since this will fail on valid but falsy values e.g. `0`

Reply:
I agree with you on the check, unfortunately this isn't due to `node.layer` being `undefined` but rather the field being non-existent in the actual data ( this is part of the discussion with kiyo where the `dataset` node would not return any data as we discussed on slack), which checking for `undefined` in this case would not capture that the `layer field` non-existent scenerio ( I had also tried modifying the test and it doesn't work.) 

that said, I had also added in the check for `undefined` to ensure a more robust test.

Reply:
That's fine, since non-existent in this case will return `undefined`.